backend/essays/module/tests/test_shared_model_package.py

```
@@ -0,0 +1,122 @@
+
+
+def _build(module) -> None:
+    """Construct the model, ignoring failures after the backbone is chosen.
+
+    The stub backbone has no real layers, so head construction raises. That is
+    past the branch under test — the recorded call is what matters.
+    """
+    try:
+        module.FilamentInstanceModelV4()
+    except Exception:  # noqa: BLE001 — see docstring
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)